src/style.css

```diff
@@ -1 +1,16 @@
 /* styles go here */
+.fix__display {
+  display: block;
+}
```

Comment:
No need to create separate class for this purpose, just apply this rule to existing class